### PR TITLE
chore(constitution): complete unified execution cutover

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Completes the remaining unified execution constitution cutover work in the
live runtime and closure surfaces. This removes live `human-only`
consequential defaults, hardens run/support-target artifacts, deletes the
flat lease compatibility file, and refreshes the active closure claim to cite
fresh April 1 ordinary live-path evidence.

## Why

The repo previously had a truth gap: closure and disclosure checks were green
while supported consequential run evidence still encoded `workflow_mode:
"human-only"` and the flat exception lease compatibility file remained live.
No-Issue: governance closeout and evidence refresh for the unified execution
constitution claim.

## How

I introduced a canonical request-binding helper in the kernel so repo-local
consequential requests always carry an active intent, actor binding, and the
bounded support-target tuple. I then hardened the authority engine and
constitutional schemas to emit and require the richer run-contract,
run-manifest, grant-bundle, and RunCard shapes, updated governance validators
to fail on live `human-only` defaults and the flat lease file, and refreshed
the active HarnessCard/closure surfaces to cite fresh April 1 agent-augmented
supported runs.

## Profile Selection Receipt

- Semantic version source(s): `.octon/framework/constitution/charter.yml`, `.octon/instance/charter/workspace.{md,yml}`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): branch-local clean-break cutover; no mixed live runtime semantics retained; fresh retained evidence emitted for supported consequential runs; rollback is commit revert or branch rollback
- Tie-break status: none
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): `n/a`

## Implementation Plan

- Workstreams and scope:
  - remove live `human-only` consequential defaults from kernel/runtime entry paths
  - require explicit support-target tuple bindings in run artifacts
  - delete the flat exception-lease compatibility file and keep only the canonical lease family
  - refresh active closure/disclosure evidence to cite fresh supported runs
  - tighten closeout validators to catch the prior overclaim conditions
- Public interfaces/types/contracts affected:
  - kernel request binding
  - authority engine run/grant/disclosure payloads
  - run-contract, run-manifest, grant-bundle, RunCard, and exception-lease schemas
- Test scenarios:
  - supported consequential workflow runs emit agent-augmented run evidence
  - closure/closeout validators remain green
  - unadmitted/unsupported adapter and pack tests continue to deny cleanly
- Assumptions/defaults:
  - bounded supported live envelope remains `MT-B / WT-2 / LT-REF / LOC-EN` on `repo-shell` + `repo-local-governed`
  - unrelated local proposal/archive edits are out of scope for this PR

## Impact Map (code, tests, docs, contracts)

- Code: kernel request construction, authority engine, runtime execution integrity, GitHub control-plane scripts/workflows
- Tests/validators: execution-governance, execution closeout, closure assertion, phase 5/6 validation, packet2 fixture lib
- Docs/disclosure: active HarnessCard, closure manifest/status matrix, clean-break release HarnessCard
- Contracts: run-contract, run-manifest, grant-bundle, RunCard, exception-lease schemas
- Evidence: fresh April 1 supported run evidence and disclosure bundles

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

Fresh retained evidence adds a large but intentional state/evidence diff because
this claim is evidence-backed and the active closure surfaces now cite the new
runs directly. Historical March artifacts remain retained for lineage, but the
active claim now points at the new supported live-path evidence.

## Testing

- `cargo check --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_kernel -p octon_authority_engine -p octon_core`
- `bash .octon/framework/assurance/governance/_ops/scripts/assert-unified-execution-closure.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-constitution-closeout.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-governance.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-disclosure-live-roots.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase5-adapter-support-target-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase6-simplification-deletion.sh`
- `git diff --check`

## Rollout

Draft PR only. This remains an atomic branch-local clean-break change set; the
rollback path is `git revert b65ecb535` or abandoning the branch before merge.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert commit `b65ecb535`
- Rollback handle: `git revert b65ecb535`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: run-contract, run-manifest, grant-bundle, RunCard, and exception-lease schemas
- Threat model update/link: validator enforcement now explicitly blocks live `human-only` consequential defaults and the flat lease compatibility path

## Observability and Performance

- Traces/logs/metrics for changed flows: fresh April 1 run evidence roots include receipts, measurements, replay pointers, and RunCards for the cited supported runs
- Representative traces for high risk changes: retained under the new April 1 run evidence roots cited by the active HarnessCard
- Performance or bundle impact: no runtime perf tuning; evidence diff size increases because fresh retained proof is now committed

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: none

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [ ] All review conversations resolved

## Screenshots / Notes

No visual change. This PR intentionally excludes unrelated local proposal/archive
moves and the pre-existing local edit to `.octon/instance/governance/contracts/closeout-reviews.yml`.

### CODE REVIEW

File: .octon/framework/engine/runtime/crates/authority_engine/src/implementation.rs [hunk 1]
```diff
@@ -1447,16 +1471,33 @@ fn stage_ref_for_request(request: &ExecutionRequest) -> String {
...
+            )
+            .with_details(json!({"reason_codes":["SUPPORT_TARGET_BINDING_MISSING"],"missing_key":key}))
+        })
+}
+
+fn requested_support_target_tuple(request: &ExecutionRequest) -> CoreResult<SupportTargetTuple> {
+    Ok(SupportTargetTuple {
+        model_tier: required_request_metadata(request, "support_model_tier")?,
+        workload_tier: required_request_metadata(request, "support_tier")?,
+        language_resource_tier: required_request_metadata(
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/engine/runtime/crates/authority_engine/src/implementation.rs:1493]: Addressed in the follow-up commits after this review snapshot. The live kernel/request constructors now bind support-target metadata up front via `.octon/framework/engine/runtime/crates/kernel/src/request.rs`, and the direct `authorize_execution()` call sites in `main.rs`, `pipeline.rs`, `workflow.rs`, and `stdio.rs` were migrated in the same branch. I intentionally kept `authority_engine` fail-closed for missing support-target bindings rather than restoring an implicit runtime fallback, because the target constitution requires explicit tuple binding on consequential paths. The remaining raw `ExecutionRequest` constructors are test fixtures inside `authority_engine`, not live runtime entrypoints.
